lib/calabash/ios/device.rb

```diff
@@ -378,6 +380,38 @@ def uninstall_app_on_physical_device(application, device_udid)
 
       private
 
+      attr_reader :runtime_info
+
+      # @!visibility private
+      def method_missing(name, *args, &block)
```

Comment:
@TobiasRoikjer 

This is the madness that I was talking about.  I am not sure yet what information will need to be extracted at runtime.  If you see the discussion section of this PR, you'll see an analysis of the info that _must_ be obtained at runtime.